lib/ObisNames.js

```diff
@@ -6,352 +6,352 @@
+            6:  "Summe Blindarbeit Q II",
+            7:  "Summe Blindarbeit Q III",
+            8:  "Summe Blindarbeit Q IV",
+            9:  "Summe Scheinleistung Bezug +",
+            10: "Summe Scheinleistung Abgabe -",
+            11: "Strom",
+            12: "Spannung",
+            13: "Leistungsfaktor",
+            14: "Frequenz",
+            16: "Gesamtwirkleistung",
+            21: "L1 Wirkarbeit Bezug +",
+            22: "L1 Wirkarbeit Abgabe -",
+            23: "L1 Blindarbeit Bezug +",
+            24: "L1 Blindarbeit Abgabe -",
+            25: "L1 Blindarbeit Q I",
+            26: "L1 Blindarbeit Q II",
+            27: "L1 Blindarbeit Q III",
+            28: "L1 Blindarbeit Q IV",
+            29: "L1 Scheinleistung Bezug +",
+            30: "L1 Scheinleistung Abgabe -",
+            31: "L1 Strom",
+            32: "L1 Spannung",
+            33: "L1 Leistungsfaktor",
+            41: "L2 Wirkarbeit Bezug +",
+            42: "L2 Wirkarbeit Abgabe -",
+            43: "L2 Blindarbeit Bezug +",
+            44: "L2 Blindarbeit Abgabe -",
+            45: "L2 Blindarbeit Q I",
+            46: "L2 Blindarbeit Q II",
+            47: "L2 Blindarbeit Q III",
+            48: "L2 Blindarbeit Q IV",
+            49: "L2 Scheinleistung Bezug +",
+            50: "L2 Scheinleistung Abgabe -",
+            51: "L2 Strom",
+            52: "L2 Spannung",
+            53: "L2 Leistungsfaktor",
+            61: "L3 Wirkarbeit Bezug +",
+            62: "L3 Wirkarbeit Abgabe -",
+            63: "L3 Blindarbeit Bezug +",
+            64: "L3 Blindarbeit Abgabe -",
+            65: "L3 Blindarbeit Q I",
+            66: "L3 Blindarbeit Q II",
+            67: "L3 Blindarbeit Q III",
+            68: "L3 Blindarbeit Q IV",
+            69: "L3 Scheinleistung Bezug +",
+            70: "L3 Scheinleistung Abgabe -",
+            71: "L3 Strom",
+            72: "L3 Spannung",
+            73: "L3 Leistungsfaktor",
+            81: "Winkel",
+            82: "Anzahl ohne Einheit",
+            91: "Neutraler Strom",
+            92: "Neutrale Spannung",
+            94: "Länderspezifisch",
+            96: "Wartungseintrag",
+            97: "Fehler",
+            98: "Listen",
+            99: "Datenprofile, Lastgang P.01/ P.02, Betriebslogbuch P.98/ P.99"
         },
         "nl": {
             0: "Algemeen doel",
-            1: "Totaal aankoop actieve energie +",
-            2: "Totaal afgegeven actieve energie -",
-            3: "Totale aankoop reactieve energie +",
-            4: "Totale reactieve energie-output -",
-            5: "Totale reactieve energie Q I",
-            6: "Totale reactieve energie Q II",
-            7: "Totale reactieve energie Q III",
-            8: "Totale reactieve energie Q IV",
-            9: "Som schijnbare stroomafname +",
-            10: "Totaal schijnbaar vermogen -",
-            11: "Actueel",
+            1: "Totale werklastenergie afname +",
```

Comment:
werklast of verbruik ?

Reply:
Ik heb geen idee! Ik weet niet wat voor een getal hier achter zit. Ik heb puur gekeken naar de Engels en Duitse woorden en de Nederlandse vertaling daarbij is 'werklastenergie'. Hetzelfde geldt voor blindlastenergie. Ik weet er niet genoeg van om te bepalen of deze vertaling hier het beste past. 

Van Google:

Elektronische energiemeters worden voornamelijk voor de registratie van **werklast**- en **blindlastenergie** gebruikt. Ze worden onder meer gebruikt voor energiemanagement en kostenplaatsenanalyse.